browser/ui/webui/playlist_ui.cc

```diff
@@ -30,9 +36,35 @@ bool PlaylistUI::ShouldBlockPlaylistWebUI(
 }
 
 PlaylistUI::PlaylistUI(content::WebUI* web_ui, const std::string& name)
-    : MojoWebUIController(web_ui) {
-  CreateAndAddWebUIDataSource(web_ui, name, kPlaylistGenerated,
-                              kPlaylistGeneratedSize, IDR_PLAYLIST_HTML);
+    : UntrustedWebUIController(web_ui) {
+  // From MojoWebUIController
+  web_ui->SetBindings(content::BINDINGS_POLICY_MOJO_WEB_UI);
+
+  auto* source =
+      CreateAndAddWebUIDataSource(web_ui, name, kPlaylistGenerated,
+                                  kPlaylistGeneratedSize, IDR_PLAYLIST_HTML);
+
+  // Allow to load untrusted resources.
+  source->OverrideContentSecurityPolicy(
```

Comment:
nit: is it possible that for the style/font/script CSP's that we can specifically reference the files we want to load in the CSP rather than anything that exists within the directory? this helps isolate things a bit further

Reply:
Oh, I yeah. Webui resources are in brave/components/playlist/resources. I guess font CSP can be removed.